curve_fitting.ipynb

```diff
@@ -155,32 +155,20 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    #i'm not sure why it is saying this error. I beoeive that I coded this right. 
</pre><p>Probably because you didn't re-execute the previous cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/4-21-22-Class/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>